pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1945,7 +1951,11 @@ class Build {
                                 if ((artifact.getRemotePath().contains(".tar.gz") || artifact.getRemotePath().contains(".zip")) && !artifact.getRemotePath().contains(".json")) {
                                     def artifactUrl = artifactoryBaseUrl + '/' + artifact.getRemotePath()
                                     def artifactName = artifactUrl.substring(artifactUrl.lastIndexOf("/") + 1);
-                                    artifactsUrls += " " + artifactUrl
+                                    if (artifactUrl.contains("-jdk_")) {
+                                        jdkUrl = artifactUrl
+                                    } else if (artifactUrl.contains("-testimage_")) {
+                                        testimageUrl = artifactUrl
+                                    }
```

Comment:
Just so I am clear, at the moment artifactsUrls contains jdk, jre, testImage and debugImage. This change will pass only jdk and for functional and openjdk targets. Is that correct? If so, can we add that to the commit comment please?

Reply:
commit comment and description are updated.